### PR TITLE
Alex quickstart readme investigation

### PR DESCRIPTION
**Pull request by alexzbusko**

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -57,9 +69,26 @@ To add support for the LIGO smart contract language, install the LIGO plugin by
 taq install @taqueria/plugin-ligo
 ```
 
-Once installed, the plugin provides the command `taq compile` which when run, will look for any LIGO files in the `/contracts` directory and compile them to Michelson `.tz` files in the `/artifacts` directory
+Once installed, the plugin provides the command `taq compile` which when run, will look for any LIGO files in the `/contracts` directory and compile them to Michelson `.tz` files in the `/artifacts` directory. The plugin also provides a template to easily create and register a Ligo contract using the `taq create contract task
```
Comment (GImbrailo) [inline quickstart.md:72]: Missing a closing "\`" at the end of `taq create contract task

File: website/docs/getting-started/quickstart.mdx [hunk 2]
```diff
@@ -87,7 +85,7 @@ To add support for the LIGO smart contract language, install the LIGO plugin by
 taq install @taqueria/plugin-ligo
 ```
 
-Once installed, the plugin provides the task `taq compile` which when run, will look for any LIGO files in the `/contracts` directory and compile them to Michelson `.tz` files in the `/artifacts` directory. The plugin also provides a template to easily create and register a Ligo contract using the `taq create contract taks
+Once installed, the plugin provides the task `taq compile` which when run, will look for any LIGO files in the `/contracts` directory and compile them to Michelson `.tz` files in the `/artifacts` directory. The plugin also provides a template to easily create and register a Ligo contract using the `taq create contract task
```
Comment (GImbrailo) [inline website/docs/getting-started/quickstart.mdx:88]: Same missing "\`" as the root `quickstart.md`

Comment (mweichert) [reply website/docs/getting-started/quickstart.mdx:88]: The compile task behaves a little different now. `taq compile` will compile any registered contracts found in the `contracts` directory, unless you run `taq compile [sourceFile]`.

**Review by GImbrailo — APPROVED** [review]
LGTM

File: taqueria-plugin-archetype/README.md [hunk 3]
```diff
@@ -24,13 +24,13 @@ No additional configuration is available
 
 The Archetype plugin exposes a `taq compile` task in Taqueria which can target one, or all the Archetype contracts in the `contracts` folder and compile them to Michelson `.tz` files output to the `artifacts` folder
 
-The Archetype plugin also exposes a contract template via the `taq create contract <contractName>` task. This task will create a new Archetype contract in the `contracts` directory, and will register the contract with Taqueria 
+The Archetype plugin also exposes a contract template via the `taq create contract <contractName>` task. This task will create a new Archetype contract in the `contracts` directory, insert archetype contract boilerplate and will register the contract with Taqueria
```
Comment (mweichert) [inline taqueria-plugin-archetype/README.md:27]: This should be `taq create archetypeContract <contractName>`

**Review by mweichert — APPROVED** [review]

**Pull request merged**